CONTRIBUTING.md

```diff
@@ -25,7 +25,9 @@ To contribute follow the next steps:
    opening one to gather feedback.
 2. Make sure to [request access](docs/adding_packages/README.md#request-access) and be aware there is a [CLA](docs/CONTRIBUTOR_LICENSE_AGREEMENT.md).
 3. Get setup by following the [Developing Recipes](docs/developing_recipes_locally.md) guide and learn the basic commands.
-4. Check the [How To Add Packages](docs/adding_packages/README.md) page for the break down of ConanCenterIndex specific conventions and practices.
+4. Check the [How To Add Packages](docs/adding_packages/README.md) page for the breakdown of ConanCenterIndex specific conventions and practices.
+   Note that not testing your new package with a specific configuration is not mandatory if you're not able to, but please do let us know
```

Comment:
I don't understand double negative wording `not testing is not mandatory` and then `if you're not able`, could you rephrase it?
does it mean `testing is mandatory if you're able`, or what exactly?